src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -307,7 +307,7 @@ def __prepareCommand(self):
             executable = shutil.which(executable) or executable
 
         # Make the full path since . is not always in the PATH
-        executable = os.path.abspath(executable)
+        executable = str(self.jobIDPath / executable)
```

Comment:
But if executable is `/bin/ls` this is going to fail, or is that treated in some other branch of the code?

Reply:
Ahh, OK, I see the Path object here works the same as os.path.join.
Thanks for adding the comment!